packages/tic-tac-toe/app/serializers/application.ts

```diff
@@ -0,0 +1,27 @@
+import DS from 'ember-data';
+import FirestoreSerializer from 'emberfire/serializers/firestore';
+import {singularize} from 'ember-inflector';
+
+export default class ApplicationSerializer extends FirestoreSerializer {
+  normalizeCreateRecordResponse(
+    _store: DS.Store,
+    _primaryModelClass: DS.Model,
+    payload: {doc: {id: string; parent: {id: string}}; data: object},
+    id: string
```

Comment:
I'm not sure I get what this does. I'm assuming that it parses the response of `this.store.createRecord` from Firebase to the client, isn't it?

Reply:
Added a comment about this.
There is an issue in latest EmberFire:
https://github.com/firebase/emberfire/pull/600